projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -17,6 +17,15 @@ export class VcdFormSelectWidgetObject extends WidgetObject<FormSelectComponent>
         return this.findElement('select').nativeElement;
     }
 
+    get clrIcon(): HTMLElement {
+        const clrIconDebugEl = this.findElement('clr-icon');
+        return clrIconDebugEl ? clrIconDebugEl.nativeElement : undefined;
+    }
+
+    findElement(selector: string | Type<unknown>, parent: DebugElement = this.root): DebugElement {
```

Comment:
It's unclear why we need this method when it just passes arguments to its super

Reply:
removed this method